examples/android/src/main/AndroidManifest.xml

```diff
@@ -2,10 +2,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.alohalytics.demoapp" >
 
-    <!-- This permission is needed if you want to access IMEI -->
-    <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
-    <!-- This one is needed to upload statistics to the server -->
+    <!-- REQUIRED to upload statistics to the server -->
```

Comment:
LGTM wrt this one then. (Although you did not answer my questions on whether they are parsed by the framework / build system / other humans.)